sdk/storage/Azure.Storage.Blobs/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/080b332b7572514a2e100dd2fa1fb86cb8edcb08/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/83ea737da20c6819531853641a432d01acbe1897/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json
```

Comment:
TODO update this when https://github.com/Azure/azure-rest-api-specs/pull/22559 merges

Reply:
Fixed.